src/ComponentFolders/TherapistCreatePage/TherapistCreatePage.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+
+function TherapistCreatePage() {
+  return (
+    <div className=" font-poppins h-screen flex flex-col justify-center content-center lg:ml-56 lg:mr-64 ml-10 mr-10 sm:ml-16 sm:mr-16 md:ml-32 md:mr-32 lg:mt-10 mt-20">
+      <h1 className="lg:mt-16 mt-4 lg:text-5xl text-xl">CREATE AN ACCOUNT</h1>
+
+      <form className="mb-0 mt-4 space-y-6" action="#" method="POST">
```

Comment:
We are not using any HTML form requests, so there is no need to write any actions or methods